mailpoet/lib/Analytics/Reporter.php

```diff
@@ -189,6 +201,17 @@ public function getData() {
     return $result;
   }
 
+  private function subscriberProperties(): array {
+    $definition = new ListingDefinition();
+    $groups = $this->subscriberListingRepository->getGroups($definition);
+    $properties = [];
+    foreach ($groups as $group) {
+      $properties['Subscribers > ' . $group['label']] = (int)$group['count'];
```

Comment:
In my opinion, you should use here a key `name` instead of `label` because the label can be translated by the user's language configuration. What do you think?